tests/python/gaia-ui-tests/gaiatest/apps/lockscreen/app.py

```diff
@@ -29,6 +29,14 @@ class LockScreen(Base):
 
     _time_locator = (By.ID, 'lockscreen-clock-time')
 
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
+        # This is to prevent the screen from turning off while on lockscreen, see bug 1169332
+        self.marionette.switch_to_frame()
+        self.marionette.execute_script("""
+            window.wrappedJSObject.ScreenManager.LOCKING_TIMEOUT = 9999;
```

Comment:
Interesting. I wonder if we should set this in GaiaTestCase.setUp to effectively disable the lockscreen timeout for all tests. Do we have any tests that rely on the lockscreen timing out? Also, there's no need to use `"""` here as we only have a single line of JS code. Just use something like:

```python
    self.marionette.execute_script(
        'window.wrappedJSObject.ScreenManager.LOCKING_TIMEOUT = 9999;')
```